backup_and_restore/application_backup_and_restore/oadp-api.adoc

```diff
@@ -0,0 +1,260 @@
+:_content-type: ASSEMBLY
+[id="oadp_api"]
+= APIs used with OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:oadp-api:
+:namespace: openshift-adp
+:local-product: OADP
+:velero-domain: velero.io
+
+toc::[]
+
+The document provides information about the following APIs that you can use with OADP:
+
+* Velero API
+* OADP API
+
+== Velero API
+
+Velero API documentation is maintained by Velero, not by Red Hat. It can be found at link:https://velero.io/docs/main/api-types/[Velero API types].
+
+== OADP API
+
+The following tables expose the structure of the OADP API:
+
+.DataProtectionApplicationSpec
+[options="header"]
+|===
+|Property|Type|Description
+
+|`backupLocations`
+|[] link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#BackupLocation[`BackupLocation`]
+|Defines the list of configurations to use for `BackupStorageLocations`.
+
+|`snapshotLocations`
+|[] link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#SnapshotLocation[`SnapshotLocation`]
+|Defines the list of configurations to use for `VolumeSnapshotLocations`.
+
+|`unsupportedOverrides`
+|map [ link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#UnsupportedImageKey[UnsupportedImageKey] ]  link:https://pkg.go.dev/builtin#string[string]
+|Can be used to override the deployed dependent images for development. Options are `veleroImageFqin`, `awsPluginImageFqin`, `openshiftPluginImageFqin`, `azurePluginImageFqin`, `gcpPluginImageFqin`, `csiPluginImageFqin`, `dataMoverImageFqin`, `resticRestoreImageFqin`, `kubevirtPluginImageFqin`, and `operator-type`.
+
+|`podAnnotations`
+|map [ link:https://pkg.go.dev/builtin#string[string] ] link:https://pkg.go.dev/builtin#string[string]
+|Used to add annotations to pods deployed by Operators.
+
+|`podDnsPolicy`
+|link:https://pkg.go.dev/k8s.io/api/core/v1#DNSPolicy[`corev1.DNSPolicy`]
```

Comment:
maybe change the link's text to just `DNSPolicy` instead of corev1.DNSPolicy, as corev1 is not an actual package (it's an import alias).